src/nitpick/style/cache.py

```diff
@@ -5,7 +5,13 @@
 from datetime import timedelta
 
 from loguru import logger
-from requests_cache.cache_control import DO_NOT_CACHE, NEVER_EXPIRE
+
+try:
+    # for requests_cache >= 1.0
+    from requests_cache.policy.expiration import DO_NOT_CACHE, NEVER_EXPIRE
```

Comment:
In my opinion, an acceptable fix would be to pin `requests_cache>=1.0` and change this code permanently.

I will try to commit to this branch.